### ISSUE TITLE
When 'd'\'u' is hold the scroll is the same speed as when holding 'j'\'k'

### ISSUE THREAD

**Issue by Vozf** [philc/vimium#3645]
**Describe the bug**

I would like to scroll faster with holding 'd' button than scrolling with holding 'j' but now it's the same speed.

**Browser and Vimium version**
Chrome

**Comment by artogahr** [comment]
Have you changed any settings recently? Can you share your custom key mappings?

**Comment by Vozf** [comment]
Nope. Custom key mappings are empty. I have smooth scrolling on. This seems to decrease the speed
